README.md

```diff
@@ -25,48 +25,54 @@ tools-deps installed.
-But for the `latest` tag it's a good trade off because for anything real we have always recommended using more specific
+But for the `latest` tag it's a good trade-off because for anything real we have always recommended using more specific
 tags. No other tags are affected by this change.
 
 ## JDK versions
 
-Java has recently introduced a new release cadence of every 6 months and dropped the leading `1` major version number.
-As of 2019-9-25, our images will default to the latest LTS release of OpenJDK (currently 11). But we also now provide
+Java follows a release cadence of every 6 months with an LTS (long-term support) release every 3 years.
+As of 2019-9-25, our images will default to the latest LTS release of OpenJDK (currently 17). But we also now provide
 the ability to specify which version of Java you'd like via Docker tags:
 
 JDK 1.8 tools-deps image: `clojure:openjdk-8-tools-deps`
 JDK 11 variant of that image: `clojure:openjdk-11-tools-deps` or `clojure:tool-deps`
-JDK 14 with the latest release of leiningen: `clojure:openjdk-14`
-JDK 15 with boot 2.8.3: `clojure:openjdk-15-boot-2.8.3`
+JDK 17 with the latest release of leiningen: `clojure:temurin-17`
+JDK 18 with boot 2.8.3: `clojure:temurin-18-boot-2.8.3`
 
 ## Linux distro
 
-The upstream OpenJDK images are built on a few different variants of Debian Linux, so we have exposed those in our
-Docker tags as well. The default is now Debian slim-bullseye. But you can also specify which distro you'd like by
-appending it to the end of your Docker tag as in the following examples (but note that not every combination is
-provided upstream and thus likewise for us):
+The upstream openjdk (java versions 11 and below) and eclipse-temurin
+(java versions 17+) images are built on a few different variants of Debian
+Linux or Ubuntu (respectively), so we have exposed those in our Docker tags as
+well. The defaults are now Debian slim-bullseye and Ubuntu focal.
+But you can also specify which distro you'd like by appending it to the end of
+your Docker tag as in the following examples (but note that not every
+combination is provided upstream and thus likewise for us):
 
-JDK 1.8 leiningen on Debian slim-bullseye: `clojure:openjdk-8` or `clojure:openjdk-8-lein` or `clojure:openjdk-8-lein-stretch`
-JDK 1.8 leiningen on Debian buster: `clojure:openjdk-8-buster` or `clojure:openjdk-8-lein-buster`
-JDK 11 tools-deps on Debian slim-bullseye: `clojure:tools-deps` or `clojure:openjdk-11-tools-deps` or `clojure:openjdk-11-tools-deps-slim-bullseye`
+Java 8 leiningen on Debian slim-bullseye: `clojure:openjdk-8` or `clojure:openjdk-8-lein` or `clojure:openjdk-8-lein-bullseye`
+Java 11 leiningen on Debian buster: `clojure:openjdk-11-bullseye` or `clojure:openjdk-11-lein-bullseye`
+Java 17 tools-deps on Debian slim-bullseye: `clojure:tools-deps` or `clojure:temurin-17-tools-deps` or `clojure:temurin-17-tools-deps-focal`
```

Comment:
did you mean "tools-deps on Eclipse Temurin:"?

Reply:
Yes! Good catch!